ambry-network/src/main/java/com.github.ambry.network/NetworkMetrics.java

```diff
@@ -78,6 +79,26 @@
   // the count of renegotiation after initial handshake done
   public final Counter sslRenegotiationCount;
 
+  // NetworkClient metrics
```

Comment:
Ah, wait for the gauges there is no problem because a new one is created when those methods are called. So they do not get overridden.